clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -74,6 +76,9 @@
     private MetadataSnapshot metadataSnapshot;
     private MetadataSnapshot assignmentSnapshot;
 
+    static final CommitFailedRetriableException RETRIABLE_COMMIT_EXCEPTION =
```

Comment:
Since this shouldn't be thrown often, I doubt storing a constant saves much overhead and we lose the stacktrace. Maybe it would be better to construct a new one each time?